sources/pmwebapi.go

```diff
@@ -0,0 +1,164 @@
+package sources
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type GetToken struct {
```

Comment:
I know for sure I've seen it flag me for having global structs without a header comment, so...not sure what the deal is.